i18n/locales/en.ts

```diff
@@ -1192,19 +1192,59 @@ const translations = {
+          'Input#0 sequence: This is the sequence for the spending input.',
+        transaction_parts_seven:
+          'Number of outputs: This indicates how many outputs the tx should have.',
+        transaction_parts_eight:
+          'Output#0 value: This is the amount that should be spent to the first output in little-endian bytes.',
+        transaction_parts_nine:
+          'Output#0 scriptPubKey: This is the public key plus OP_CHECKSIG to ensure that the transaction is being spent correctly.',
+        transaction_parts_ten:
+          'Output#1 value: This is the amount that should be spent to the second output in little-endian bytes.',
+        transaction_parts_eleven:
+          'Output#1 scriptPubKey: This is the public key of the sender so that the change output will go back into their possesion.',
+        transaction_parts_twelve:
+          'Locktime: Amount of time that this tx should be locked before the coins can be spent.',
+      },
+      derive_message_seven: {
+        sighash_type_flag_heading: 'SigHash Type Flag',
+        sighash_type_flag_paragraph_one:
+          'SIGHASH flags are a mechanism in Bitcoin that define which parts of a transaction are included in the hash that is signed by a private key. Essentially, they determine the scope of commitment by the signer to specific parts of the transaction data. The SIGHASH flag is a single byte appended to each signature and can vary between inputs within the same transaction. There are several types of SigHash flags you can learn about <Link href="https://river.com/learn/terms/s/sighash-flag" target="_blank" className="underline">here</Link>.',
       },
       verify_signature_two: {
         tip_one:
           'JavaScript hint: You can convert a hex string to a buffer of bytes using <span className="p-1 font-mono bg-[#00000033] m-1">Buffer.from(someString, \'hex\');</span>',
+        signature_verification_heading: 'Signature Verification',
+        signature_verification_paragraph_one:
+          'Signature verification is crucial in Bitcoin to confirm that a transaction is authorized by the holder of the private key. In the context of ECDSA, it involves checking that a signature (comprising two numbers, r and s) is valid for a given public key and message. This verification ensures the integrity and authenticity of a transaction.',
+      },
+      verify_signature_three: {
+        signature_encoding_heading: 'Signature Encoding',
+        signature_encoding_paragraph_one:
+          'A (DER) signature or Distinguished Encoding Rules is simply a formate used to encode an ECDSA signature in Bitcoin. An ECDSA signature is generated using a private key and a hash of the signed message. It consists of two 32-byte numbers (r,s). It has multiple components you can learn more about <Link href="https://bitcoin.stackexchange.com/questions/12554/why-the-signature-is-always-65-13232-bytes-long/12556#12556" target="_blank" className="underline">here</Link>.',
```

Comment:
```suggestion
          'A (DER) signature or Distinguished Encoding Rules is simply a format used to encode an ECDSA signature in Bitcoin. An ECDSA signature is generated using a private key and a hash of the signed message. It consists of two 32-byte numbers (r,s). It has multiple components you can learn more about <Link href="https://bitcoin.stackexchange.com/questions/12554/why-the-signature-is-always-65-13232-bytes-long/12556#12556" target="_blank" className="underline">here</Link>.',
```

Reply:
This is a pretty good not bad resource on DER signatures in Bitcoin :wink: https://technicaldifficulties.io/2020/07/22/bip-66-unpacking-der-signatures/

Matthew even gets a shoutout in it for catching an error in a previous revision :smile: